mlos_core/mlos_core/tests/optimizers/optimizer_multiobj_test.py

```diff
@@ -93,39 +95,41 @@ def objective(point: pd.DataFrame) -> pd.DataFrame:
         optimizer.get_observations()
 
     for _ in range(max_iterations):
-        suggestion, metadata = optimizer.suggest()
-        assert isinstance(suggestion, pd.DataFrame)
-        assert metadata is None or isinstance(metadata, pd.DataFrame)
-        assert set(suggestion.columns) == {"x", "y"}
+        suggestion = optimizer.suggest()
+        assert isinstance(suggestion, Suggestion)
+        assert isinstance(suggestion.config, pd.Series)
+        assert suggestion.metadata is None or isinstance(suggestion.metadata, pd.Series)
+        assert set(suggestion.config.index) == {"x", "y"}
         # Check suggestion values are the expected dtype
-        assert isinstance(suggestion.x.iloc[0], np.integer)
-        assert isinstance(suggestion.y.iloc[0], np.floating)
+        assert isinstance(suggestion.config["x"], int)  # type: ignore
+        assert isinstance(suggestion.config["y"], float)  # type: ignore
```

Comment:
Does `np.integer` not work here anymore?  Would be nice to drop the type ignore (or at least improve it's specificity)